src/Tribe/RSVP.php

```diff
@@ -1194,106 +1194,26 @@ public function generate_tickets( $post_id = null, $redirect = true ) {
 	 *
 	 * @since 4.5.2 added $event_id parameter
 	 * @since 5.5.10 Adjusted the method to use the new Tickets Emails Handler.
+	 * @since TBD Reverted the methods back to before 5.5.10, new Tickets Emails Handler via filters.
 	 *
 	 * @param int $order_id The order ID.
 	 * @param int $event_id The event ID.
 	 */
 	public function send_tickets_email( $order_id, $event_id = null ) {
-		if ( ! tec_tickets_emails_is_enabled() ) {
-			return $this->send_tickets_email_legacy( $order_id, $event_id );
-		}
-
-		$all_attendees = $this->get_attendees_by_order_id( $order_id );
-
-		$to_send = [];
-
-		if ( empty( $all_attendees ) ) {
-			return;
-		}
-
-		// Look at each attendee and check if a ticket was sent: in each case where a ticket
-		// has not yet been sent we should a) send the ticket out by email and b) record the
-		// fact it was sent.
-		foreach ( $all_attendees as $single_attendee ) {
-			// Do not add those attendees/tickets marked as not attending (note that despite the name
-			// 'qr_ticket_id', this key is not QR code specific, it's simply the attendee post ID).
-			$going_status = get_post_meta( $single_attendee['qr_ticket_id'], self::ATTENDEE_RSVP_KEY, true );
-			if ( in_array( $going_status, $this->get_statuses_by_action( 'count_not_going' ), true ) ) {
-				continue;
-			}
-
-			// Only add those attendees/tickets that haven't already been sent.
-			if ( empty( $single_attendee['ticket_sent'] ) ) {
-				$to_send[] = $single_attendee;
-			}
-		}
-
 		/**
-		 * Controls the list of tickets which will be emailed out.
+		 * @since TBD
 		 *
-		 * @param array $to_send        list of tickets to be sent out by email
-		 * @param array $all_attendees  list of all attendees/tickets, including those already sent out
-		 * @param int   $order_id
+		 * @param null|mixed $pre      Determine if we should continue.
+		 * @param int        $order_id The order ID.
+		 * @param int        $event_id The event ID.
+		 * @param static     $module   Instance of the Tickets Module.
 		 */
-		$to_send = (array) apply_filters( 'tribe_tickets_rsvp_tickets_to_send', $to_send, $all_attendees, $order_id );
+		$pre = apply_filters( 'tec_tickets_send_rsvp_email_pre', null, $order_id, $event_id, $this );
```

Comment:
So once you deprecate the old email sending, you'll have the email sending function with only a filter named `pre_something` where's no `something`?

I think will create a lot of confusion when deprecating old emails (as well as now), and it is something that we want to do soon. That's why we had a different approach. It would be better if we talked about these things, so you can have more context instead of creating PRs and merging them immediately.